### PR TITLE
Model loading

### PR DESCRIPTION
**Pull request by tigranfah**
* add llama configs for training
* add the possibility of downsampling some molecular properties during the training
* add special tokens for the property tags, smiles and safe
* add CONFORMER tag to the tokenizer
* verify logits matching with huggingface when download and exporting llama

### CODE REVIEW

**Comment by MenuaB** [comment]
LGTM

**Review by MenuaB — APPROVED** [review]
